src/test/java/teammates/test/cases/webapi/GetOngoingSessionsActionTest.java

```diff
@@ -22,8 +30,103 @@ protected String getRequestMethod() {
 
     @Override
     @Test
-    protected void testExecute() {
-        // TODO
+    protected void testExecute() throws Exception {
+        ______TS("Verify no ongoing session");
+
+        String[] params = new String[] {
+                Const.ParamsNames.FEEDBACK_SESSION_STARTTIME, "0",
+                Const.ParamsNames.FEEDBACK_SESSION_ENDTIME, "1000",
+        };
+
+        GetOngoingSessionsAction getOngoingSessionsAction = getAction(params);
+        JsonResult r = getJsonResult(getOngoingSessionsAction);
+
+        verifyNoExistingSession(r);
+
+        ______TS("Typical use case");
+
+        InstructorAttributes instructor1OfCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        String courseId = instructor1OfCourse1.courseId;
+
+        Instant startTime = Instant.now();
+        Instant endTime = Instant.now().plus(5, ChronoUnit.DAYS);
+
+        logic.createFeedbackSession(FeedbackSessionAttributes.builder("new-session",
```

Comment:
Personally, I don't like your coding style for builder chain call.
```
FeedbackSessionAttributes.builder(....)
        .withXXX()
        .withXXX()
        .build()
```
is prefered. In addition, do break line at high level.
```
logic.createFeedbackSession(
        FeedbackSessionAttributes.builder("new-session", courseId, instructor1OfCourse1.email)
```
this is preferred.